gcc/testsuite/rust.test/xfail_compile/unsafe.rs

```diff
@@ -0,0 +1,4 @@
+fn main() { // { dg-ice "Segmentation fault" }
```

Comment:
To cross-reference, would it be useful to use `dg-ice "#382"` (or similar)?

Reply:
Hmmm, maybe yes. If the string is informational, "Segmentation fault" is not really a good pick. I thought the string was matched after the 'Internal compiler error' (so that you could check you are ICEing in a specific function), but I did not really read the doc and simply (badly it seems) inferred from examples